### PR TITLE
Fix package configuration for release

### PR DESCRIPTION
**Pull request by truh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +35,8 @@
     ]
   },
   "dependencies": {
-    "@types/markdown-it": "10.0.2",
-    "markdown-it": "11.0.0"
+    "@types/markdown-it": "^10.0.2",
+    "markdown-it": "^11.0.0"
```
Comment (mrdrogdrog) [inline package.json:39]: Why did you readd the version ranges?

File: package.json [hunk 2]
```diff
@@ -44,6 +47,6 @@
     "jest": "26.4.2",
     "punycode": "2.1.1",
     "ts-jest": "26.2.0",
-    "typescript": "4.0.2"
+    "typescript": "3.9.7"
```
Comment (mrdrogdrog) [inline package.json:50]: Why did you downgrade to ts 3?

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (truh) [reply package.json:50]: ts-jest doesn't like v4 yet.

File: package.json (hunk 1 above)
Comment (truh) [reply package.json:39]: It's not a good idea to have fixed versions in npm packages as it forces npm to install two different versions of the same package if multiple packages have common dependencies.

This is not only a waste of space but also causes builds to break as soon as typescript is involved.

Comment (mrdrogdrog) [reply package.json:39]: When we installed renovate bot we read https://docs.renovatebot.com/dependency-pinning/#pinning-dependencies-and-lock-files which explains when you should pin your dependencies and when you should not.
TL;DR: If you are a dependency, then you should keep semver ranges. This is a dependency, so it's okay for me.

**Review by mrdrogdrog — APPROVED** [review]

**Comment by mrdrogdrog** [comment]
(Please write a PR description. Even if it's short)

**Review by ErikMichelson — APPROVED** [review]
